### PR TITLE
2 group based collection àla twacapic

### LINKED ISSUE #2: Group-based collection àla twacapic
For on-going collections a mechanism for collecting account groups would be good.
I.e. a call to `tegracli get account1 account2` could receive an additional option to save the resulting list to a subfolder from which we can update and/or rerun the collection: `tegracli get --name_group my_accountlist account1 account2 ...`.
Hence, that would allow for a call in the like of `tegracli update my_accountlist`. A setup like this could have folder structure similar to this:
```
folder/
  tegracli.conf.yml
  my_accountlist/
    12932xxx.jsonl
    14204xxx.jsonl
```
Optionally, we should introduce some verbs to manipulate these accountlists, i.e. `tegracli remove $account1` to remove accounts from the list, as well as `add` to add accounts to a list.

### PR DESCRIPTION
**Pull request by pekasen**
This feature-PR adds the ability to collect posts and update the collection on a rolling basis from a account list.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,6 +100,62 @@ Options:
   --help  Show this message and exit.
 ```
 
+## GROUP INIT and GROUP RUN
+
+In order to support updatable  and long running collections `tegracli` sports an *account group* feature which retrieves
+the history of a given set of accounts and is able to retrieve updates on each of these accounts.
+
+Groups are initialized by calling `teracli group init`, where accounts to track are stated by either stating them as arguments
```
Comment (FlxVctr) [inline README.md:108]:
```suggestion
Groups are initialized by calling `tegracli group init`, where accounts to track are stated by either stating them as arguments
```

File: tests/test_cli.py [hunk 2]
```diff
@@ -50,26 +64,132 @@ def test_configuration_present(runner: CliRunner, tmp_path: Path):
...
-    result = runner.invoke(cli, ["search", "term"])
+    For this it must create a directory named by user input, read in a file with tg-accounts,
+    resolve those user names and save messages by user in a jsonl-file.
 
-    assert result.exit_code == 0
+    I.e. a call to this command would look like this:
+    `tegracli group init --read_file account_list.csv my_little_account_list` to
+    load a account list from a file and write the configuration file to disk.
+    """
+    with runner.isolated_filesystem(temp_dir=tmp_path) as temp_dir:
```
Comment (FlxVctr) [inline tests/test_cli.py:91]: nice

File: tegracli/utilities.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+import datetime
+from functools import singledispatch
+from typing import Any, Dict, Union
+
+from telethon import TelegramClient
+
+from .types import AuthenticationHandler
+
+
+@singledispatch
```
Comment (FlxVctr) [inline tegracli/utilities.py:12]: this is super fancy, but will be hard to read/maintain for most junior programmers/students we will hire in the future

**Review by FlxVctr — APPROVED** [review]
Again, too large a PR to review in-depth. Your tests look better every day though :)

However, group.py and dispatch.py could need some more coverage, especially for the edge cases that are not covered yet (e.g. retesting unreachable accounts, errors etc.)

**Comment by FlxVctr** [comment]
Also, heaps of deprecation warnings when testing and a loguru error
